practice.kt

```diff
@@ -6,18 +6,22 @@ fun main() {
  }
  fun printRectangularFrameList(list:List<String>){
      var longestElement:Int = 0
-     for (element in list){
+
+    for (element in list){
```

Comment:
It is not indented. It should like;

for (element in list){
          if(longestElement<element.length)
              longestElement = element.length
     }